convert_types.go

```diff
@@ -0,0 +1,8 @@
+package datadog
+
+func Int(val interface{}) *int {
```

Comment:
I originally started coding up the same implementation, but if your function is named Int, has as single purpose returning a pointer to an int (or it returns nil), why not change the signature to only accept an int like so?:

```go
func Int(v int) *int { return &v }
```

(See https://github.com/zorkian/go-datadog-api/pull/83/files#diff-01127be9ed6b6352616a2f1c10bc3540R29 )